### DISCUSSION TITLE
[IDEAS NEEDED] Style About page

### DISCUSSION THREAD

**Discussion by amandamartin-dev** [EddieHubCommunity/EddieHubCommunity.github.io#323] [Ideas]
I think we should style the About page and make it look a bit more like the about page rather than what it is right now

Please suggest ways in which we can style the page

**Comment by nilerudra** [comment]
I suggest focusing on several key elements to enrich the "About" page:

- Can share journey, milestones, and the values that define EddieHub. A compelling narrative can resonate deeply with audience.

- Highlighting team members of EddieHub and their roles can humanize our brand, fostering trust and relatability.

- Articulating  mission, vision, and core values reaffirms purpose and principles of EddieHub to the audience.

- Showcasing community engagement efforts, success stories, and impact can demonstrate commitment to making a difference.

**Reply by ikrishg** [reply]
hey @nilerudra! I've converted this into a discussion! Since you were assigned earlier, we will consider you a priority for assignment whenever this issue is ready to be worked on! However, if you'd like to contribute there are several other issues available

**Reply by nilerudra** [reply]
I appreciate being considered a priority for this assignment. I'm definitely up for contributing!

**Comment by ikrishg** [comment]
I agree with @nilerudra's suggestions, I am keen on having something like (list is in order as it would be on page):

- About EddieHub
- Core Values [I can provide content, I remember the core values)
- Mission/Vision
- Numbers (possibly automated, such as GitHub organization member count, discord member count)
- Testimonials (Reuse the component on home page)
- Big header saying "Our Team"
- Founders section (I don't know if there are founders other than @eddiejaoude, so Eddie'll have to tell us)
- Moderators (move that section entirely from the home page)
- Maintainers section (cc #99)

I am keen on reusing some of these sections on the home page to account for the moving of the moderators' section:

- Core Values
- Numbers

**Reply by SaraJaoude** [reply]
Taking into consideration @nilerudra comments - it is important to reflect that EddieHub is **not** a company and should not be mentioned as such anywhere on this page or the site in general. 

@xkrishguptaa:

- As to Core Values I don't think you need to provide any new content on this as we should be using the wording that is already on the page. 

As to the list you mention this seems to relate to making changes to the entire site (e.g. Moderators page, Testimonials etc) - this Discussion refers to the About page so I suggest we keep it focused on this for now.  Also any changes should be done in small sections as it is easier to review.
